Backend/app/rider-platform/rider-app/Main/spec/Storage/sos.yaml

```diff
@@ -20,7 +20,7 @@ Sos:
       enum: "EmergencyContactId Text"
 
     SosType:
-      enum: "Police,CustomerCare,EmergencyContact EmergencyContactId,SafetyFlow,PoliceAPI"
```

Comment:
Check this for backward compatibility,

Reply:
No entry present anywhere. Since it was never enabled.